linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/http/IstioIdentifier.scala

```diff
@@ -0,0 +1,80 @@
+package io.buoyant.linkerd.protocol.http
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.twitter.finagle.Stack.Params
+import com.twitter.finagle.buoyant.Dst
+import com.twitter.finagle.{Dtab, Path}
+import com.twitter.finagle.http.Request
+import com.twitter.finagle.param.Label
+import com.twitter.util.Future
+import io.buoyant.config.types.Port
+import io.buoyant.k8s.{ClientConfig, IstioPilotClient, RouteManager}
+import io.buoyant.linkerd.IdentifierInitializer
+import io.buoyant.linkerd.protocol.HttpIdentifierConfig
+import io.buoyant.router.RoutingFactory.{IdentifiedRequest, Identifier, RequestIdentification, UnidentifiedRequest}
+import com.twitter.conversions.time._
+import istio.proxy.v1.config.RouteRule
+
+class IstioIdentifier(pfx: Path, baseDtab: () => Dtab, routeManager: RouteManager) extends Identifier[Request] {
+  private[this] val unidentified: RequestIdentification[Request] =
```

Comment:
Should we fail the request if no routes match?  Or forward the request to it's stated destination (I _think_ this is what istio does) ?  I was thinking we could use `/svc/dest/<hostname>/<port>` in the case where no rules match.